dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/DefaultDataSetMetadataExportService.java

```diff
@@ -158,6 +166,46 @@ private Set<CategoryOption> getCategoryOptions(
         return options;
     }
 
+    /**
+     * Returns data sets as a list of {@link ObjectNode}. Includes associations
+     * to organisation units.
+     *
+     * @param dataSets the list of {@link DataSet}.
+     * @param dataSetOrgUnits the associations between data sets and
+     *        organisation units.
+     * @return data sets as a list of {@link ObjectNode}
+     */
+    private List<ObjectNode> getDataSets( List<DataSet> dataSets, SetValuedMap<String, String> dataSetOrgUnits )
```

Comment:
`get` is maybe an overused prefix for a method doing a conversion or creation of a new type of object from another object. I suggest `create` or `to` as prefixes in such cases.

Reply:
Agreed. Fixed.